docs/user/quick-start/installation/system-requirements.md

```diff
@@ -0,0 +1,29 @@
+---
+title: System Requirements
+summary: Hardware Requirements Guide
+---
+
+# System Requirements
+
+- A blank DVD or a USB drive that is at least 3GB
+- On UEFI systems, an EFI partition of 1GB minimum
+- An internet connection
+- System connected to AC power
+
+In order to install Solus, your system must meet the minimum system requirements. Ideally it should meet or exceed the recommended values for a better experience.
+
+|                       | Minimum                         | Recommended                                                                               |
+| --------------------- | ------------------------------- | ----------------------------------------------------------------------------------------- |
+| **Disk space free**       | 10GB / 9.3GiB                   | 25GB / 23.3GiB                                                                            |
+| **Memory (RAM)**          | 4GB                             | 8GB or more                                                                               |
+| **Processor (CPU)**       | 64-bit (x86_64) processor       | [x86-64-v2](https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels) 2GHz quad core |
+| **System firmware**       | BIOS / UEFI                     | UEFI                                                                                      |
+| **Display Adapter (GPU)** | Any onboard or dedicated adapter |                                                                                           |
+
+These are general recommendations. Your requirements may vary based on your intended use of the system.
+
+:::info
+
```

Comment:
The info box is a little vague. We should remove it or provide details on which cards we support, IMO

Reply:
I can see this. I had inspiration from the [Fedora requirements](https://docs.fedoraproject.org/en-US/fedora/latest/release-notes/welcome/Hardware_Overview/) for the wording. I've added the list they have for GPUs that are old enough to probably not support acceleration. Is this better?

Reply:
That's perfect